e2e/tests/test_snapshotjob.py

```
@@ -0,0 +1,635 @@
+    # Overrides the conftest.py `run` fixture for this module: SnapshotJob
+    # cleanup is shaped differently (delete the SnapshotJob, not a bare
+    # PodSnapshot by run.snapshot_name — see cleanup_snapshotjob's docstring).
+    value = snap.TestRun.new(request.node.name.replace("_", "-")[:24])
+    yield value
+    snap.cleanup_snapshotjob(config, value)
+
+
+def assert_snapshotjob_completed(sj: dict[str, Any]) -> None:
+    """Asserts the full success vector of the two-stage completion gate."""
+    completed = snap.condition(sj, "Completed")
+    assert completed and completed.get("reason") == "JobCompleted"
```

Ruff (0.16.2)

[warning] Assertion should be broken down into multiple parts

Break down assertion into multiple parts

(PT018)